std_msgs/Header header
int32[] original_size
int32[] strings_shape
BytesList[] strings
